draft-ietf-jsonpath-base.md

```diff
@@ -396,9 +396,9 @@ of bracket notation. Examples and descriptions use shorthands where convenient.
 
 ### Selectors
 
-A name selector, e.g. `'name'`, selects a named child of an object.
+A name selector, e.g. `'name'`, selects a named child in an object.
```

Comment:
We can talk about a named value in an object, but when it comes to children, I would say that an object's children are children _of_ the object, not children _in_ the object. Similarly for arrays.

Reply:
Unborn child?  (Just kidding.)
+1

Reply:
Ok:
```suggestion
A name selector, e.g. `'name'`, selects a named child of an object.
```

Reply:
Since "of" includes "in" (the keyboard *of* my laptop), I think we can accept this suggestion.